site/mkdocs.yml

```diff
@@ -3,7 +3,7 @@ strict: false
 site_name: Fybrik
 site_description: A cloud-native platform to control data usage
 copyright: Made with ❤️ by Fybrik contributors.
-repo_name: fybrik/fybrik
+repo_name: fybrik.io/fybrik
```

Comment:
Is the organization going to be called `fybrik.io`? I thought it would be just fybrik

Reply:
This is the name that appears at the top right of the site which links to the github repo.